flink-java/src/main/java/org/apache/flink/api/java/DataSet.java

```diff
@@ -1337,11 +1336,16 @@ public long count() throws Exception {
 	/**
 	 * Writes a DataSet to the standard output stream (stdout).<br/>
 	 * For each element of the DataSet the result of {@link Object#toString()} is written.
```

Comment:
The javadoc should contain a note that print() is triggering the execution of the job.